src/parser/expression.js

```diff
@@ -352,31 +355,31 @@ pp.parseCallExpressionArguments = function (close, possibleAsyncArrow) {
   }
 
   return elts;
-};
+}
 
-pp.shouldParseAsyncArrow = function () {
+shouldParseAsyncArrow(): boolean {
```

Comment:
For a function declared in `parserTypes.ts`, the return type *should* be carried along from there. But for a function declared only locally, you may get no (or confusing) errors. I think it's better to just consistently declare types. E.g., remove the return type from `parseCallExpressionArguments`, add `elts.push(0);` somewhere, and you won't get any errors.